matchzoo/datapack.py

```diff
@@ -65,14 +76,30 @@ def relation(self):
         return self._relation
 
     @property
-    def content(self):
-        """Get :meth:`content` of :class:`DataPack`."""
-        return self._content
-
-    @content.setter
-    def content(self, value: dict):
-        """Set the value of :attr:`content`."""
-        self._content = value
+    def left_data(self):
+        """Get :meth:`left_data` of :class:`DataPack`."""
+        return self._left_data
+
+    @left_data.setter
+    def left_data(self, value: pd.DataFrame):
```

Comment:
same, it's wired to use `left_data` as property, and user will never remember this property by name.